apps/server/src/modules/course-synchronization-history/repo/course-synchronization-history.repo.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { CourseSynchronizationHistory } from '../do';
```

Comment:
Need to be placed in domain layer. 
But i see the folder structure need also a litte change.

repo -> domain <- api (see dependency order of the layer) domain stuff can use in repo und api
domain = service | do | the repo interface